src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/AbstractKubernetesPipelineRJRTest.java

```diff
@@ -18,7 +18,16 @@ public abstract class AbstractKubernetesPipelineRJRTest {
     public TestName name = new TestName();
 
     @Rule
-    public RealJenkinsRule rjr = new RealJenkinsRule();
+    public RealJenkinsRule rjr;
+    {
+        rjr = new RealJenkinsRule();
+        String port = System.getProperty("port");
```

Comment:
Where would this get set? https://github.com/jenkinsci/kubernetes-plugin/blob/87f08371da37a00e3be16c6b039699de06c2959d/test-in-k8s.sh#L33 is set in the Maven JVM, not passed along to the Surefire booter currently: https://github.com/jenkinsci/kubernetes-plugin/blob/87f08371da37a00e3be16c6b039699de06c2959d/pom.xml#L268-L276

Reply:
For reference, did you figure this out? I see https://ci.jenkins.io/job/Plugins/job/kubernetes-plugin/job/PR-1290/6/testReport/org.csanchez.jenkins.plugins.kubernetes.pipeline/KubernetesPipelineRJRTest/kind___basicPipeline/ passing with

```
Overriding port using system property: 7221
```

but it is unclear to me how it works.